src/model/PlanDefinition.ts

```diff
@@ -91,12 +91,13 @@ export default class PlanDefinition implements IPlanDefinition {
                         year = dobNumbers[0];
                     }
                     if (dobNumbers.length >= 2) {
-                        month = dobNumbers[1];
+                        const dobMonth =  dobNumbers[1];
+                        // in a Date object, month is 0-index based, 0 = Jan, 1 = Feb
+                        month = dobMonth > 0 ? dobMonth-1 : dobMonth;
```

Comment:
Doesn't this result in Jan and Feb both being 0?

Reply:
@mcjustin sorry, my comment there is confusing, I have since fixed it to clarify.
so dobMonth here would be 2 (for February), we need to decrease it to 1 when we pass it to the Date object.  Same for the case of 1 (for January), which we need to decrease to 0 before passing it to the Date object.
In this case, Jan is 0, Feb is 1.

Reply:
@mcjustin I don't believe a 0 can be entered on the UI for the birthdate month (due to validation check).  So it will be very unlikely that it will ever be 0.